source/class/cv/ui/PageHandler.js

```diff
@@ -84,8 +84,6 @@ qx.Class.define('cv.ui.PageHandler', {
 
       templateEngine.currentPage = pageWidget;
 
-      qx.event.message.Bus.dispatchByName("path."+target+".duringPageChange", target);
```

Comment:
I deleted is for two reasons:

1. No one is using it
2. There is no "during" page change. Because Javascript is single-threaded you can only do something before the page changes or after it changed. If you count in animations we might find a use case for for doing something when the page gets visible (and the animation starts). You can already listen to an "changeVisible" event on a page object to capture that moment. 
But it might be arguable to introduce a new bus message for this, when there is a need for it.

So long story short. As soon as there is a need for it I'm open to adding another bus event, but currently its not needed.